model/core/callbacks.sail

```diff
@@ -7,6 +7,12 @@
 // =======================================================================================
 
 // Callbacks for state-changing events
+
+// Called upon successful instruction fetch with the opcode.
+// If there is a fetch error then this is not called (but trap_callback() will be).
```

Comment:
Perhaps we should start using the doc-comment '///' prefix for these kinds of comments in case we want to extract them to documentation later (here and below in this file)?